### PR TITLE
pending changes workflow

### PR DESCRIPTION
**Pull request by ssickles**
updated the cli commands for apply, plan, and up to work with the new workspace_changes

### CODE REVIEW

File: runs/set_module_version.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package runs
+
+import (
+	"fmt"
+	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+)
+
+func SetModuleVersion(cfg api.Config, workspace types.Workspace, version string) (*types.WorkspaceChangeset, error) {
```
Comment (BSick7) [inline runs/set_module_version.go:9]: The usage of this function seems to use module@version as the version input. Does the API expect only the version?

**Review by BSick7 — APPROVED** [review]
One thing to check. Looks good.

File: runs/set_module_version.go (hunk 1 above)
Comment (ssickles) [reply runs/set_module_version.go:9]: I built the API endpoint thinking that we would only allow the user to change the module version (thus the name). I ended up relying on the runConfigResolver to perform this override. I went back in and realized that the runConfigResolver actually allows both the module and the version to be overridden. Thus I am passing this in with the full module@version. I'm trying to move quick and not overthink. Should I rename the endpoint or change the runConfigResolver to just accept a version?

Comment (BSick7) [reply runs/set_module_version.go:9]: 👍 Awesome, I just wanted to confirm there wasn't a miscommunication between the user and API.
I think renaming the endpoint would be ideal.

**Pull request merged**